schemas/follow_up.json

```diff
@@ -131,12 +131,12 @@
       "meta": {
         "core": true,
         "dependsOn": "follow_up.disease_status_at_followup",
-        "notes": "This field is required to be submitted if disease_status_at_followup indicates progression or relapse.",
+        "notes": "This field is required to be submitted if disease_status_at_followup indicates progression or relapse or recurrence.",
```

Comment:
would it be more grammatically correct to say `
-  indicates a state of progression, relapse, or recurrence`

Reply:
same comments for fields below

Reply:
Hmm, I thought I had fixed the script validation but maybe I missed this one. You're right, it should be required if it's "relapse or recurrence". I'll correct this.

Reply:
@rosibaj I fixed this.